docker/package/tezos_baking_wizard.py

```diff
@@ -0,0 +1,466 @@
+#!/usr/bin/env python3
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+"""
+A wizard utility to help set up tezos-baker.
+
+Asks questions, validates answers, and executes the appropriate steps using the final configuration.
+"""
+
+import os, sys, subprocess, shlex, shutil
+import readline
+import re, textwrap
+import urllib.request
+from typing import List
+
+# Input validators
+
+
+def enum_range_validator(options):
+    def _validator(input):
+        intrange = list(map(str, range(1, len(options) + 1)))
+        if not input in intrange:
+            raise ValueError(
+                "Please choose one of the provided values: " + ", ".join(intrange)
+            )
+        return list(options.keys())[int(input) - 1]
+
+    return _validator
+
+
+def filepath_validator(input):
+    if input and not os.path.isfile(input):
+        raise ValueError("Please input a valid file path.")
+    return input
+
+
+def required_field_validator(input):
+    if not input.strip():
+        raise ValueError("Please provide this required option.")
+    return input
+
+
+class Validator:
+    def __init__(self, validator):
+        self.validator = validator
+
+    def validate(self, input):
+        if self.validator is not None:
+            if isinstance(self.validator, list):
+                for v in self.validator:
+                    input = v(input)
+                return input
+            else:
+                return self.validator(input)
+
+
+# Wizard CLI utility
+
+
+def proc_call(cmd):
+    return subprocess.check_call(shlex.split(cmd))
+
+
+def fetch_snapshot(url):
+    print("Downloading the snapshot from ", url)
+    filename = "tezos_node.snapshot"
```

Comment:
Looks like it will download the snapshot to the current directory (from which this wizard is called), thus `tezos-node` won't have sufficient rights to use this file, I suggest downloading it to the `/tmp`
```suggestion
    filename = "/tmp/tezos_node.snapshot"
```